core/src/main/resources/jenkins/fingerprints/GlobalFingerprintConfiguration/config.properties

```diff
@@ -0,0 +1,24 @@
+# The MIT License
+#
+# Copyright (c) 2004-, Kohsuke Kawaguchi, Sun Microsystems, Inc., and a number of other of contributors
```

Comment:
Common practice is to put your copyright, or to just reference Jenkins contributors as suggested below

```suggestion
# Copyright (c) 2022- Jenkins contributors
```